root.tf

```diff
@@ -142,12 +142,14 @@ module "terraform_state_lock" {
   common_tags = local.common_tags
 }
 
-//Set up common IAM policies for Terraform
-module "terraform_permissions" {
+//Set up common IAM policies for Terraform and for publishing to S3
+module "common_permissions" {
   source                 = "./modules/permissions"
   common_tags            = local.common_tags
   terraform_state_bucket = module.terraform_state.terraform_state_bucket_arn
   terraform_state_lock   = module.terraform_state_lock.terraform_state_lock_arn
+  release_bucket_arn              = module.release_artefacts_s3.s3_bucket_arn
```

Comment:
Nitpick: alignment.

Reply:
I ran terraform fmt. It was happy about the indentation in common_permissions so I've left that one.